graviteeio/0/docker-compose.yml

```diff
@@ -0,0 +1,82 @@
+#-------------------------------------------------------------------------------
+# Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#            http://www.apache.org/licenses/LICENSE-2.0
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#-------------------------------------------------------------------------------
+
+elasticsearch:
```

Comment:
Please add ulimit to avoid 'too many open files'
```
    ulimits:
      nofile: 65536
```
see https://github.com/gravitee-io/gravitee-docker/commit/05f698c383280215374a038aefb186c102db449c